File: Aulas/Aula3/Retangulo.go
package main

import "fmt"

// Definindo a estrutura Retangulo
type Retangulo struct {
	comprimento, largura float64
}

// Função para calcular a área do retângulo
func (r Retangulo) calcularArea() float64 {
	return r.comprimento * r.largura
}

// Função para calcular o perímetro do retângulo
func (r Retangulo) calcularPerimetro() float64 {
	return 2 * (r.comprimento + r.largura)
}

func main() {
	// Criando uma instância da estrutura Retangulo
	ret := Retangulo{comprimento: 200, largura: 300}
	fmt.Printf("Área: %.2f\n", ret.calcularArea())
	fmt.Printf("Perímetro: %.2f\n", ret.calcularPerimetro())
}

File: Aulas/Aula3/Retangulo.java
public class Retangulo {
    private double comprimento;
    private double largura;

    // Construtor
    public Retangulo(double comprimento, double largura) {
        this.comprimento = comprimento;
        this.largura = largura;
    }

    // Método para calcular a área
    public double calcularArea() {
        return comprimento * largura;
    }

    // Método para calcular o perímetro
    public double calcularPerimetro() {
        return 2 * (comprimento + largura);
    }

    public static void main(String[] args) {
        // Criando uma instância da classe Retangulo
        Retangulo ret = new Retangulo(200, 300);
        System.out.println("Área: " + ret.calcularArea());
        System.out.println("Perímetro: " + ret.calcularPerimetro());
    }
}

